### ISSUE TITLE
Rename reafactor_release_build job

### ISSUE THREAD

**Issue by karianna** [AdoptOpenJDK/openjdk-build#1097]
remove https://ci.adoptopenjdk.net/view/all/job/openjdk_release_tool/ and rename https://ci.adoptopenjdk.net/view/all/job/refactor_openjdk_release_tool/ 

We will need to update the name in the build scripts though

**Comment by hendrikebbers** [comment]
@M-Davies @karianna  I assume that this can only be done by having admin rights on Jenkins? Next to this both links are not working. Is this issue still valid?

**Comment by karianna** [comment]
Yes requires admin access which is probably why you can't see them, not a good first issue
